src/broadcast.ts

```diff
@@ -3,11 +3,15 @@ import * as _ from 'lodash'
 import {RippleAPI} from './api'
 
 class RippleAPIBroadcast extends RippleAPI {
+
+  // TODO: Should this default to 0, or null/undefined?
```

Comment:
I think `0` is good here.

Reply:
```
> 1 > undefined
false
```

Reply:
@intelliot my concern is that 0 is a number but isn't a valid ledger number. It do mean that things like `(ledger.ledgerVersion > this.ledgerVersion)` will work by default, but it also means we have to document and explain that 0 = unset/unknown. It would be more explicit if `ledgerVersion = undefined` means it hasn't been looked up yet.

Reply:
That's a good point. I'm fine with `undefined`, but make sure to update the if condition (`ledger.ledgerVersion > this.ledgerVersion`) too.